ui/tests/integration/components/pki/pki-role-form-test.js

```diff
@@ -118,7 +118,7 @@ module('Integration | Component | pki-role-form', function (hooks) {
     const groupBoxHeight = find('[data-test-toggle-div="Key usage"]').clientHeight;
     assert.strictEqual(
       groupBoxHeight,
```

Comment:
Issue currently with failing test. Need to revisit. Locally 572 passes, on PR runs its 573.